webapp/src/main/kotlin/webapp/ports/NotificationController.kt

```diff
@@ -0,0 +1,39 @@
+package webapp.ports
+
+import domain.Status
+import domain.actions.ChangeStatusAction
+import domain.asDeviceId
+import domain.asThingId
+import domain.messages.ChangeStatusMessage
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/v1/notification")
+class NotificationController(
+    private val changeStatusAction: ChangeStatusAction
+) {
+
+    @PostMapping("/change-status")
+    fun changeStatus(
+        @RequestBody request: ChangeStatusRequest
+    ): ResponseEntity<Any> {
+        changeStatusAction.changeStatus(request.toMessage())
+        return ResponseEntity.noContent().build()
+    }
+}
+
+data class ChangeStatusRequest(
```

Comment:
MAJOR: How can the device send the device id and thing id if it just knows the id on device (1, 2, etc..) ?